bindings/java/src/main/java/org/openucx/jucx/ucs/UcsConstants.java

```diff
@@ -22,6 +22,58 @@
         public static int UCS_THREAD_MODE_MULTI;
     }
 
+    /**
+     * Status codes
+     */
+    public enum STATUS {
+        /* Operation completed successfully */
+        UCS_OK                        (0),
+
+        /* Operation is queued and still in progress */
+        UCS_INPROGRESS                (1),
+
+        /* Failure codes */
+        UCS_ERR_NO_MESSAGE            (-1),
```

Comment:
can we avoid setting value for each? like we do with UCP API flags for example?
`static long UCP_PARAM_FIELD_FEATURES;`